### PR TITLE
Removed company index page and route

### PR DESCRIPTION
**Pull request by mlgiardina**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
   authenticate :employee do
 
     resources :rooms
-    resources :companies
+    resources :companies, except: :index
```
Comment (JMD1986) [inline config/routes.rb:11]: good idea.

File: app/views/companies/index.html.erb [hunk 2]
```diff
@@ -1,25 +0,0 @@
...
-        <td><%= link_to 'Edit', edit_company_path(company) %></td>
-        <td><%= link_to 'Destroy', company, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Company', new_company_path %>
```
Comment (JMD1986) [inline app/views/companies/index.html.erb:25]: :put_litter_in_its_place:

**Comment by mlgiardina** [comment]
working on getting the tests working

**Comment by mlgiardina** [comment]
i removed the tests that were associated with the company index

**Comment by jah2488** [comment]
:+1:

**Pull request merged**